cli/src/helpers/local/commentHelper.ts

```diff
@@ -96,17 +99,22 @@ export async function getDashboard(req: express.Request, res: express.Response)
 
         if (fs.existsSync(absolutePath)) {
           const indexPath = path.join(absolutePath, 'index.html');
-
           if (fs.lstatSync(absolutePath).isFile()) {
             linkUpdate = await runServersAndGetLinkUpdate(
               staticServers,
               taskId,
               localPath,
               path.dirname(absolutePath),
-              path.basename(absolutePath)
+              (port) => `http://localhost:${port}/${path.basename(absolutePath)}`
             );
           } else if (fs.existsSync(indexPath)) {
```

Comment:
File viewers won't show up in local mode for now because the stoat server can't reach localhost (the api call is made server-side).

Maybe that should change to client-side or maybe it's fine to not access this locally?